packages/bitcore-node/src/providers/chain-state/internal/internal.ts

```diff
@@ -224,6 +224,19 @@ export class InternalStateProvider implements CSP.IChainStateService {
     Storage.apiStreamingFind(CoinModel, query, stream);
   }
 
+  async getFee(params: CSP.GetEstimateSmartFeeParams) {
```

Comment:
Let's go ahead and use the asyncCall method :)

Reply:
resolved